action.yml

```diff
@@ -1,34 +1,35 @@
 name: 'deploy-infrastructure'
 author: '1up-team'
-description: 'Action to plan and deploy Terraform modules'
+description: 'Plan/deploy Terraform stacks'
 
 inputs:
   service-name:
-    description: 'Service name, used to derive Terraform state file on backend'
+    description: 'Used to derive Terraform state file on backend: ecs_service_{service-name}'
     required: true
   module-path:
-    description: 'Absolute path to module, e.g. ./terraform, ./terraform/vdfproxy-foobar'
+    description: 'Relative path to the Terraform stack dir'
     required: false
     default: 'terraform'
 
   pkg-token:
-    description: 'User token used to integrate with the spring-media GitHub packages for internal libs'
+    description: 'spring-media GitHub packages token. Used for calling the GitHub API and by Terraform for cloning 1up-infrastructure modules'
     required: true
 
   docker-image-tag:
```

Comment:
Since we need the docker tag here, we could move the ecs-deploy step in here as well.
I think we would need another action name if we do that.